app/views/posts/show.html.erb

```diff
@@ -1,12 +1,15 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1>Here is a selected post with for a designated user.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div id="post">
+  <h3><%= @post.title %> by <%= @post.author.name %></h3>
+  <p class="counter">
+    Comments: <%= @post.comments_counter.to_i %>, Likes: <%= @post.likes_counter.to_i %>
+  </p>
+  <p><%= @post.text %></p>
+</div>
+
+<ul id="comment-container">
+  <% @post.comments.each do |comment| %>
+    <li id="comment">
+      <p><%= comment.user.name %>: <%= comment.text %></p>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] **[OPTIONAL]** Kindly minimize the space of the containers in this page. The height of the containers should be relative to the content, like the design.

  ![image](https://github.com/katarighe/blog-app/assets/81468675/6c4c63a6-a3e8-428c-abb9-ade2357fa216)